config/locales/geoblacklight.en.yml

```diff
@@ -2,8 +2,6 @@ en:
   geoblacklight:
     bbox_label: 'Bounding Box'
     location: 'Location'
-    citation:
```

Comment:
Surprised we dropping the "Retrieved from " preface here? — Isn't that APA style? AND... nope, looks like APA 7th drops the "Retrieved from" requirement in citations.

Reply:
Thanks. 

1. Class added to fix close alert alignement.
2. Copy to clipboard was broken due a missing translation (which added a span, which broke the javascript).  Fixed.